pkg/installation/installation.go

```diff
@@ -50,12 +57,15 @@ func New(ctx context.Context, fromUrl string) (*Installation, error) {
 		return nil, microerror.Mask(err)
 	}
 
+	baseEndpoint := strings.Split(info.Kubernetes.ApiUrl, urlDelimiter)[1:]
```

Comment:
you can use the `basePath` variable already defined 👍🏻

Reply:
🙈